services/wercker/wercker.service.js

```diff
@@ -15,53 +15,6 @@ const werckerSchema = Joi.array()
   .required()
 
 module.exports = class Wercker extends BaseJsonService {
-  static getBaseUrl({ projectId, applicationName }) {
-    if (applicationName) {
-      return `https://app.wercker.com/api/v3/applications/${applicationName}/builds`
-    } else {
-      return `https://app.wercker.com/api/v3/runs?applicationId=${projectId}`
-    }
-  }
-
-  async fetch({ baseUrl, branch }) {
-    return this._requestJson({
-      schema: werckerSchema,
-      url: baseUrl,
-      options: {
-        qs: {
-          branch,
-          limit: 1,
-        },
-      },
-      errorMessages: {
-        401: 'private application not supported',
-        404: 'application not found',
-      },
-    })
-  }
-
-  static render({ result }) {
-    return renderBuildStatusBadge({ status: result })
-  }
-
-  async handle({ projectId, applicationName, branch }) {
-    const json = await this.fetch({
-      baseUrl: this.constructor.getBaseUrl({
-        projectId,
-        applicationName,
-      }),
-      branch,
-    })
-    if (json.length === 0) {
-      return this.constructor.render({
-        result: 'not built',
-      })
-    }
-    const { result } = json[0]
-    return this.constructor.render({ result })
-  }
-
-  // Metadata
   static get category() {
     return 'build'
   }
```

Comment:
Side note on the route below... that is one helluva `format` 😄 Should be a whole lot of fun converting that to `pattern`

Reply:
Indeed! Thank goodness for redirects.